setup.py

```diff
@@ -39,6 +39,16 @@
             'share/users',
         ]),
         ('lib/systemd/system', ['packaging/temboard-agent.service']),
+        # Installation from sources
+        ('/usr/share/temboard-agent/', [
```

Comment:
This breaks installation is userspace (e.v. `pip install --user` or in virtualenv).

Reply:
Cf. http://readthedocs.org/projects/temboard-agent/builds/6440200/

Reply:
Thank you. How to ship them properly then ?

Reply:
Alright, we do not need absolute path when installing from sources using `pip install .`. Let's go that way.